src/components/Hero.tsx

```diff
@@ -0,0 +1,38 @@
+import "../_styles/index.scss";

```

Comment:
Do we need to import styles here? It already done in App.tsx.

Reply:
- [ ] Done.